Configure

```diff
@@ -756,11 +756,22 @@ while (@argvcopy)
 			$disabled{"shared"} = "forced";
 			$disabled{"threads"} = "forced";
 			}
-		elsif (/^-D(.*)$/)
+		elsif (/^[-\/]D(.*)$/)
 			{
 			push @user_defines, $1;
 			}
-		else	# common if (/^[-+]/), just pass down...
+		elsif (/^\/link$/)
+			{
+			# This is Visual Studio-specific thing.
+			# Consume all options that start with slash, e.g.
+			# /link /subsystem:console,5.1. Yes, it has to
+			# be last or terminated with non-slash option.
+			while (@argvcopy[0] =~ /^\//)
+				{
+				$libs.=shift(@argvcopy) . " ";
+				}
+			}
```

Comment:
Oh, we will definitely have to keep the old way in parallell for 1.1.x, I agree with that.